### ISSUE TITLE
Need reviewers for managing issues and pull requests

### ISSUE THREAD

**Issue by dostonnabotov** [dostonnabotov/quicksnip#30]
Hey, everyone!

As more people contributing to our project (which is awesome), it will be kinda harder for me to maintain and review the code snippets. Therefore, I'm looking for reviewers to review the code snippets to make sure they are actually helpful for developers.

I also noticed that @Mathys-Gasnier is already reviewing a lot of code snippets and finding out issues with them. Thank you very much. Appreciate your work. I wanted to ask: Is it okay if I assign you, @Mathys-Gasnier, as a reviewer for our project. Will you have available time to review? Let me know.

If anybody is willing to be part of reviewing and helping manage the issues and pull requests, let me know in this issue. 

I'm not very familiar with some languages, like C++ and Rust, therefore, @saminjay was also helping me out as well. Thank you. Once we have some reviewers, we can divide the roles based on their expertise. 

As always, I'm open to ideas. Let me know your thoughts.

**Comment by psychlone77** [comment]
I could help out with reviewing some PRs. Mostly JavaScript, Python and CSS stuff will be possible.

**Comment by saminjay** [comment]
I would love to help out with reviewing PRs and managing Issues.
I am mostly familiar with JS, C++, Go and Python.

**Comment by Mathys-Gasnier** [comment]
Yeah it's not a problem !
I have a lot of time (Will probably have a bit less in a couple of weeks, but still more than 5-6 hours a day) so reviewing stuff is not an issue and i really like the project.
Maybe creating a discord would be something interesting btw, to discuss stuff without having to open an issue for every little things.

On what languages i'm familiar with it's mostly, javascript/typescript, java, python and low level languages mainly C/C++ and Rust.
But i've worked a bit with C# and some other techs not long ago, so i'm pretty versatile.

**Comment by realvishalrana** [comment]
Hi @dostonnabotov ,

I would love to help as a reviewer for this project. I have experience in JavaScript, React, Next.js, and CSS. Please let me know how I can assist!

Best,
Vishal Rana

**Comment by dostonnabotov** [comment]
Thanks to everyone who is willing to be part of this project. To maintain trust and collaboration with reviewers, first, we will need to prove that you can contribute to the project well. As you contribute more, the more responsibility you will have. Hopefully, with this approach, we will build trusted maintainers for this awesome community.

I also noticed that @Yugveer06, @GreenMan36 and @saminjay are heavily focusing on improving the code and adding new features. And, @Mathys-Gasnier is responsible for reviewing the pull requests. Special thanks to you all!

**Comment by dostonnabotov** [comment]
I just found out that there's a thing called CODEOWNERS, which you assign people to specific folders or files. When there's a pull request with change on that folder, assigned people will be notified and they need to review it before getting merged. 

What do you think about it, @Mathys-Gasnier? 

I'm open to other ideas as well.

**Comment by Mathys-Gasnier** [comment]
I think we could have two types of reviewers, global ones, that review changes for the project and for snippets, and snippets reviewers that are assigned one or more language to review snippets from

**Comment by realvishalrana** [comment]
@Mathys-Gasnier I completely agree with this suggestion. Having two types of reviewers—global reviewers for overall project changes and snippets reviewers dedicated to specific languages—would streamline the review process. It ensures that domain experts can focus on language-specific snippets, while global reviewers oversee broader changes. This division of responsibility could significantly enhance the efficiency and quality of reviews. Let me know how I can contribute to implementing this!

**Comment by dostonnabotov** [comment]
> Maybe creating a discord would be something interesting btw, to discuss stuff without having to open an issue for every little things.

I like the idea. What do you think about GitHub Discussions? I'm not much familiar with it yet, so let me know which one would be easier to maintain in the long run.

**Comment by Mathys-Gasnier** [comment]
I have lot's of experience with discord, and not much with github discussions, what i know is that most libs I contribute to have a discord... If you want we can always set it up together if you don't know how to create one

**Comment by dostonnabotov** [comment]
Ok. I will create a Discord server and share the link in a new issue.

**Comment by ACR1209** [comment]
Hey, are you still looking for reviewers? If so, I would be down to help out!  

I'm familiar with Python, Rust, JS, TS, Ruby and Haskell.

**Comment by pl44t** [comment]
If you are still looking for reviewers, I would be happy to help with python, css, html, javascript.

**Comment by Mathys-Gasnier** [comment]
Hey, thanks all for volunteering on being reviewers.
Feel free to leave reviews on PRs if you want to give your opinion on new snippets/snippets changes !
If we see a PR has been getting lots of approval from the community it will better help us to decide if we should add it or not.

Like @dostonnabotov said, if we see you are cotributing a lot (adding snippets, reviewing and giving feedback) we will be able to talk with you to maybe make you a reviewer of certains snippets languages !

**Comment by Kol9yN** [comment]
I can review Bash, Vagrant(VirtualBox, VmWare), Ansible code snippets and add basic templates of it. Maybe something more a bit later, lets see how the going :) @dostonnabotov
